spec2vec/Spec2Vec.py

```diff
@@ -1,16 +1,73 @@
+from typing import Union
 import scipy
+from spec2vec import SpectrumDocument
 from .calc_vector import calc_vector
 
 
 class Spec2Vec:
+    """Calculate spec2vec similarity scores between a reference and a query.
 
-    def __init__(self, model, intensity_weighting_power=0):
+    Using a trained model, spectrum documents will be converted into spectrum
+    vectors. The spec2vec similarity is then the cosine similarity score between
+    two spectrum vectors.
+
+    Example code to calcualte spec2vec similarities between query and reference
+    spectrums:
+
+        import gensim
+        from spec2vec import Spec2Vec
+        from spec2vec import SpectrumDocument
+
+        references = [SpectrumDocument(s, n_decimals=2) for s in reference_spectrums]
```

Comment:
Can you specify where reference_spectrums & query_spectrums  come from.
```suggestion
      
        # reference_spectrums & query_spectrums loaded from files using https://matchms.readthedocs.io/en/latest/api/matchms.importing.load_from_mgf.html 
        references = [SpectrumDocument(s, n_decimals=2) for s in reference_spectrums]
```